lib/topological_inventory/ansible_tower/parser/service_instance.rb

```diff
@@ -3,14 +3,16 @@ class Parser
     module ServiceInstance
       def parse_service_instance(job_hash)
         job = job_hash[:job]
+
+        external_url = "#{self.tower_host}#{job.url}"
```

Comment:
Depending on what the url looks like something like a URI.join or a File.join is probably a little safer so we don't assume a leading or trailing slash